Cannot start strong-studio, missing module

Following instructions in https://github.com/strongloop/strong-studio when I ran `strong-studio` it fails with `Error: Cannot find module 'loopback-explorer'`.  This was with new project created with slc.

```
$ strong-studio

module.js:340
    throw err;
    ^
Error: Cannot find module 'loopback-explorer'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/usr/local/lib/node_modules/strong-studio/server/server.js:4:16)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
```

--- a/server/server.js
+++ b/server/server.js
@@ -1,7 +1,6 @@
 var express = require('express');
 var path = require('path');
 var workspace = require('loopback-workspace');
-var explorer = require('loopback-explorer');
 
 var app = module.exports = express();
 
@@ -10,6 +9,7 @@ app.use('/workspace', workspace);
 
 try {
   // API explorer
+  var explorer = require('loopback-explorer');
   app.use('/explorer', explorer(workspace, { basePath: '/workspace/api' }));
 } catch(err) {
   // silently ignore the error, the explorer is not available in "production"

